io.openems.edge.controller.ess.predictivedelaycharge/src/io/openems/edge/controller/ess/predictivedelaycharge/ac/Config.java

```diff
@@ -0,0 +1,30 @@
+package io.openems.edge.controller.ess.predictivedelaycharge.ac;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition( //
+		name = "Controller Predictive Delay Charge AC", //
+		description = "controller which calculates the power limitations every second to optimize the charging from PV")
```

Comment:
Not quite well explaining what this Controller actually does and why there are two versions of it...